extensions/slack/package.json

```diff
@@ -69,13 +70,40 @@
       "type": "checkbox",
       "required": false,
       "default": true
+    },
+    {
+      "title": "Time Format",
+      "label": "Time Format",
+      "description": "Determines how time is formatted in the extension",
+      "name": "timeFormat",
+      "type": "dropdown",
+      "required": false,
+      "data": [
+        {
+          "value": "hour12",
+          "title": "12-hour format (AM/PM)"
+        },
+        {
+          "value": "hour24",
+          "title": "24-hour format"
+        }
+      ],
+      "default": "hour12"
+    },
+    {
+      "title": "Search Command",
+      "label": "Display extra metadata in search results",
+      "description": "Show team member's titles, time zone, and current time in search results",
+      "name": "displayExtraMetadata",
```

Comment:
Question (non-blocking): shouldn't this be added to command preferences rather than global preferences?

Reply:
Woah, didn't realize you could do that. Update for that coming soon 
![Raycast-Settings_2024-07-07 at 20 34@2x](https://github.com/raycast/extensions/assets/1895120/9834a872-9b30-48c7-a62f-381633f1c643)